runner/src/test/java/com/redhat/developer/runner/it/End2EndTestIT.java

```diff
@@ -0,0 +1,149 @@
+package com.redhat.developer.runner.it;
+
+import java.net.URI;
+import java.time.Duration;
+
+import org.awaitility.Awaitility;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.developer.infra.api.APIConstants;
+import com.redhat.developer.infra.dto.BridgeStatus;
+import com.redhat.developer.infra.utils.CloudEventUtils;
+import com.redhat.developer.manager.api.models.requests.BridgeRequest;
+import com.redhat.developer.manager.api.models.responses.BridgeListResponse;
+import com.redhat.developer.manager.api.models.responses.BridgeResponse;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+
+import static io.restassured.RestAssured.given;
+
+@QuarkusTest
```

Comment:
Can you use [`@QuarkusIntegrationTest`](https://quarkus.io/guides/getting-started-testing#quarkus-integration-test) annotation? It should be the best fit for integration (and it should work for native too if necessary).

Reply:
@danielezonca I've tried that. But it's like the annotation builds the artifact and runs it with `java -jar ...`: I did not find a way to inject the configurations.

tbh I did not spend much time on that, so it's probably possible, but I'd suggest to stay with @QuarkusTest that works pretty well at least until we are using the All-In-One application.  https://quarkus.io/guides/getting-started-testing#quarkus-integration-test

Reply:
Ok fine for me 👍 
We can revisit in the future while looking at native compilation